internal/go118_sha1_patch.go

```diff
@@ -0,0 +1,47 @@
+package internal
```

Comment:
This is put into a separate file so that it can be called by unit tests who can't import the `command` package.